brv/importer/svcomp.py

```diff
@@ -17,7 +17,7 @@ def extractXmlLinks(email):
 
 
 def extractVerifierName(email):
-    regex = re.compile(r'Please find below the (final|preliminary) results for verifier ([^,]+),')
+    regex = re.compile(r'Please find below the (final|Pre-run) results for tool ([^,]+),')
```

Comment:
What about loading of last years results? I think you should extend the expression, not change it.

Reply:
Good idea!